dispers/src/jvmMain/kotlin/dev/schlaubi/lavakord/dispers/Builder.kt

```diff
@@ -0,0 +1,29 @@
+package dev.schlaubi.lavakord.dispers
+
+import dev.bitflow.dispers.client.DispersClient
+import dev.schlaubi.lavakord.LavaKord
+import dev.schlaubi.lavakord.LavaKordOptions
+import dev.schlaubi.lavakord.MutableLavaKordOptions
+import kotlin.properties.Delegates
+
+/**
+ * Creates a [LavaKord] instance for this [Kord] instance.
+ *
+ * @param configure a receiver configuring the [LavaKordOptions] instance used for configuration of this instance
+ */
+public fun DispersClient.lavakord(configure: DispersLavaKordOptions.() -> Unit): LavaKord {
+    val dispersOpt = DispersLavaKordOptions().apply(configure)
+    val options = MutableLavaKordOptions().apply(dispersOpt.lavaKordOptions).seal()
+    return DispersLavaKord(
+        this,
+        dispersOpt.botId,
+        dispersOpt.botShards,
+        options
+    )
+}
+
+public class DispersLavaKordOptions {
+    public var botId: ULong by Delegates.notNull()
+    public var botShards: Int by Delegates.notNull()
```

Comment:
Is there no options to read this from the dispers instance, so there is no need to configure it manually?

Reply:
The dispers instance is technically bot acgnostic and thus this must be handled seperately. Dispers works fully on an on-event basis in terms of context (bot id, shard id, etc).